easybuild/easyconfigs/n/nsync/nsync-1.24.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeNinja'
+
+name = 'nsync'
+version = '1.24.0'
+
+homepage = 'https://github.com/google/nsync'
+description = """FlatBuffers: Memory Efficient Serialization Library """
```

Comment:
Should be nsync description instead of FlatBuffers

Reply:
Thanks!